### ISSUE TITLE
All new LORAs produce "modules.devices.NansException: A tensor with all NaNs was produced in Unet" in StableDiffusion

### ISSUE THREAD

**Issue by ssbugman** [kohya-ss/sd-scripts#1091]
Was using very old version, replaced with fresh install to be able to train SDXL and all new LORA become broken.
Reducing learning rate or training steps to very low allows to generate but only on level where model not actually trained yet and not giving anything close to dataset - so its useless anyway.
